src/components/binding/binding-renderer.js

```diff
@@ -725,7 +773,7 @@ genotet.BindingRenderer.prototype.drawBed_ = function() {
       return rectWidth < this.BED_MIN_WIDTH ? this.BED_MIN_WIDTH :
         rectWidth;
     }.bind(this))
-    .attr('height', this.bedRectHeight_ > 0 ? this.bedRectHeight_ : 0)
+    .attr('height', Math.max(this.bedRectHeight_, this.BED_MIN_HEIGHT))
```

Comment:
The rendering code should avoid computing size values. this.bedRectHeight_ can actually be the final height applied and can be computed preferrably in layout().